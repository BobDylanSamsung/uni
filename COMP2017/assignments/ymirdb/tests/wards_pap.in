set a 1 2 3
set b 4 5 6
set c 9 8 7 0
set d 0
push b a
append c b 2 a
push d a c
list entries
forward a
forward b
forward c
forward d
backward a
backward b
backward c
backward d
del d
list entries
pluck c 5
forward c
backward b
backward a
bye
