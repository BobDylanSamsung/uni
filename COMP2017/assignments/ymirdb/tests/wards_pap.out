ok

ok

ok

ok

ok

ok

ok

d [c a 0]
c [9 8 7 0 b 2 a]
b [a 4 5 6]
a [1 2 3]

nil

a

a, b

a, b, c

b, c, d

c, d

d

nil

ok

c [9 8 7 0 b 2 a]
b [a 4 5 6]
a [1 2 3]

b

a

nil

b, c

bye
